### PR TITLE
ISSUE_TEMPLATE: diploma example for reputation

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Thanks for your interest in telling the story of RChain in another language.
     - [ ] at least two of them have high proficiency in (the target language)
     - [ ] at least two of them have high proficiency in English
     - [ ] at least two of them have either
-       - [ ] significant reputation at stake; source: ____
+       - [ ] significant reputation ( (e.g. a diploma) at stake; source: ____
```
Comment (dckc) [inline ISSUE_TEMPLATE.md]: parens are mis-matched

**Review by dckc — CHANGES_REQUESTED** [review]

**Comment by dckc** [comment]
also, the title of a PR is too important for generic stuff like "Update ISSUE_TEMPLATE.md". Say "give diploma as concrete example of credentials" or the like

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (ICA3DaR5) [reply ISSUE_TEMPLATE.md]: @dckc Fixed.

**Comment by ICA3DaR5** [comment]
@dckc You are right.

**Review by dckc — APPROVED** [review]

**Pull request merged**